feature-libs/customer-ticketing/styles/components/_customer-ticketing-list.scss

```diff
@@ -0,0 +1,76 @@
+cx-customer-ticketing-list {
```

Comment:
You should be using placeholders selectors. An example can be found here https://github.com/SAP/spartacus/pull/16044/commits/5cb1d7acaf03997ab209878961e2cfa8fe056a42.

An explaination can be found here https://github.com/SAP/spartacus/pull/16044#discussion_r941734729

Reply:
Awesome will change this!